### PR TITLE
More translations and blacklisted and additional nuclei templates

### PR DESCRIPTION
**Pull request by kazet**

### CODE REVIEW

File: artemis/reporting/modules/nuclei/translations/nuclei_messages/pl_PL.py [hunk 1]
```diff
@@ -347,6 +347,7 @@
     "The Openstack host is configured as a proxy which allows access to the instance metadata service. This could allow significant access to the host/infrastructure.": "Wykryto, że serwer HTTP jest skonfigurowany jako serwer proxy umożliwiający dostęp do wewnętrznych metadanych OpenStack, w tym potencjalnie do haseł dostępowych.",
     "The AWS host is configured as a proxy which allows access to the metadata service. This could allow significant access to the host/infrastructure.": "Wykryto, że serwer HTTP jest skonfigurowany jako serwer proxy umożliwiający dostęp do wewnętrznych metadanych AWS, w tym potencjalnie do haseł dostępowych.",
     "[no description] http/exposures/files/ds-store-file.yaml": "Wykryto plik .DS_Store, zawierający informację o nazwach plików w katalogu, w tym potencjalnie np. kopii zapasowych lub innych plików, które nie powinny być publicznie dostępne.",
+    "[no description] http/misconfiguration/server-status-localhost.yaml": "Wykryto, że końcówka /server-status serwera Apache jest publicznie dostępna, udostępniając takie informacje jak np. konfigurację serwera, adresy IP użytkowników czy odwiedzane przez nich strony.",
```
Comment (szymsid) [inline artemis/reporting/modules/nuclei/translations/nuclei_messages/pl_PL.py:350]: shouldn't there be another entry for `"[no description] http/misconfiguration/server-status.yaml"`

File: artemis/reporting/modules/nuclei/translations/nuclei_messages/pl_PL.py [hunk 2]
```diff
@@ -425,4 +426,6 @@
     "Hestia Control Panel login was detected.": "wykryto panel logowania narzędzia Hestia Control Panel.",
     "Kibana login panel was detected.": "wykryto panel logowania narzędzia Kibana.",
     "Check for the existence of the ArcGIS Token Service on an ArcGIS server.": "wykryto narzędzie ArcGIS Token Service.",
+    "The jUDDI (Java Universal Description, Discovery and Integration) Registry is a core component of the JBoss Enterprise SOA Platform. It is the product's default service registry and comes included as part of the product. In it are stored the addresses (end-point references) of all the services connected to the Enterprise Service Bus. It was implemented in JAXR and conforms to the UDDI specifications.": "wykryto narzędzie jUDDI (Java Universal Description, Discovery and Integration) Registry.",
+    'Check for the existence of the "/arcgis/rest/services" path on an ArcGIS server.': "wykryto API narzędzia ArcGIS.",
```
Comment (szymsid) [inline artemis/reporting/modules/nuclei/translations/nuclei_messages/pl_PL.py:430]: did we already detect this, but didn't have the translations?

Comment (kazet) [reply artemis/reporting/modules/nuclei/translations/nuclei_messages/pl_PL.py:430]: yes - I add translations only after a template actually finds something

File: artemis/reporting/modules/nuclei/translations/nuclei_messages/pl_PL.py (hunk 1 above)
Comment (kazet) [reply artemis/reporting/modules/nuclei/translations/nuclei_messages/pl_PL.py:350]: I decided to add translations only after a template actually finds something

**Review by szymsid — APPROVED** [review]

**Pull request merged**